scripter/scripter.go

```diff
@@ -45,15 +49,18 @@ type ConnectionWrapper interface {
 	Handle(message string) (string, error)
 	SetStringFunction(name string, getString func() string) error
 	SetFloatFunction(name string, getFloat func() float64) error
+	SetVoidFunction(name string, doVoid func()) error
 	GetParameter(index int) (string, error)
 }
 
 type ScrConn interface {
+	GetConn() net.Conn
+	GetAbTester() abtester.Abtester
```

Comment:
I do think that AB tester must not be in the SrcConn interface, this way you will force people to use it.
Make it a seperate interface and with a call, check if the conn is a ABTester interface

Reply:
OK!